spring-kafka/src/test/java/org/springframework/kafka/retrytopic/RetryTopicBootstrapperTests.java

```diff
@@ -66,12 +66,14 @@
 
 	@Test
 	void shouldThrowIfACDoesntImplementInterfaces() {
-		assertThrows(IllegalStateException.class, () -> new RetryTopicBootstrapper(wrongApplicationContext, beanFactory));
+		assertThatExceptionOfType(IllegalStateException.class)
+				.isThrownBy(() -> new RetryTopicBootstrapper(wrongApplicationContext, beanFactory));
 	}
```

Comment:
We prefer to use `assertThatIllegalStateException()` and `assertThatIllegalArgumentException()` for these.

Reply:
@garyrussell Fixed.